ui/client/src/components/PaladinTransaction.tsx

```diff
@@ -45,10 +45,12 @@ export const PaladinTransaction: React.FC<Props> = ({ paladinTransaction }) => {
   }
 
   const formatProperty = (value: any) => {
```

Comment:
Is this used anywhere else? Can be a util function if so

Reply:
Good point, this is a good candidate to be moved into an utilities file. I also took the opportunity to rename it and adjust the formatting, so now JSON structures are easier to read:

<img width="600" alt="image" src="https://github.com/user-attachments/assets/7c855828-7d83-42be-a64c-1f99c5eed2e2" />